docs/jsonapi-spec.textile

```diff
@@ -233,12 +235,24 @@ bc(sample)..
     } 
 } 
 
+h3(#commandFindOne). Find One Command
+
+h3(#commandFindOneAndUpdate). Find One And Update Command
+
+h3(#commandInsertMany). Insert Many Command
+
+h3(#commandInsertOne). Insert One Command
+
+h3(#commandUpdateMany). Update Many Command
```

Comment:
this does not work, ends up in the `#update-many-command`..

See here: https://github.com/stargate/jsonapi/blob/92eeb73cb951e5d199bbc35e0223d2084c009ca9/docs/jsonapi-spec.textile#update-many-command

Reply:
Ahh seems that both anchors work in fact, also tried now and it works: https://github.com/stargate/jsonapi/blob/92eeb73cb951e5d199bbc35e0223d2084c009ca9/docs/jsonapi-spec.textile#commandUpdateMany